### DISCUSSION TITLE
Rabbitmq 3.6.8 ，conection flow，nodedown

### DISCUSSION THREAD

**Discussion by c327812419** [rabbitmq/rabbitmq-server#6379] [Other] [answered]
Sorry, my English is not very good.
I'm using rabbitmq 3.6.8.
Rabbitmq suddenly stop working. Here are some screenshots.

connection flow:
![connection flow](https://user-images.githubusercontent.com/8746339/200606164-c4f8ffc1-b697-49d1-948d-72987f96538a.png)

disk space and memery is low
![hard memery](https://user-images.githubusercontent.com/8746339/200606416-6fcbc70a-338c-46ba-9846-b9209231c02a.png)

rabbitmqctl status  
![nodedonw](https://user-images.githubusercontent.com/8746339/200606626-837cd6d5-159b-47bf-9044-45afb6f180bb.jpg)

log
![mqlog](https://user-images.githubusercontent.com/8746339/200606683-7bf4741d-9e17-457f-a0de-ae2b7fc95135.png)

Consumer utilisation is 100% (only use sms_*)
![consumer 100](https://user-images.githubusercontent.com/8746339/200606995-74f4dab2-52c3-418e-b62c-b3d35515618c.jpg)

netstat 
![netstat](https://user-images.githubusercontent.com/8746339/200607077-e059e54b-d210-424e-a49c-d48604b2eed1.png)

mangement
![manage](https://user-images.githubusercontent.com/8746339/200607156-711a5820-5ea9-4639-b4a5-b8843d82805a.png)

top
![top](https://user-images.githubusercontent.com/8746339/200607191-b09ec50f-5336-4764-bbf5-0f9a91aec9c3.png)

What's the problem? thanks a lot.

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ 3.6 has [reached end of life](https://www.rabbitmq.com/versions.html) in May 2018.
Please [upgrade](https://www.rabbitmq.com/upgrade.html), we no longer provide any guidance for versions that are out of support (older than 3.9).

**Reply by c327812419** [reply]
Thanks a lot. I'm going to upgrade.
